### ISSUE TITLE
Sketcher: Minor Layout Issues

### ISSUE THREAD

**Issue by bruderol** [scenarioo/scenarioo#805]
When details are hidden:

![image](https://user-images.githubusercontent.com/3780183/40849819-3ba64ba6-65c3-11e8-8818-1f37545566de.png)

On a small screen ("responsive"):

![image](https://user-images.githubusercontent.com/3780183/40849849-5384d77e-65c3-11e8-8e97-c61a8dc144d9.png)

as reported by @adiherzog

**Comment by bruderol** [comment]
sorry, not needed. will close this duplicate of #717 again. we can still split it once a part of it is done.
